aws/elasticbeanstalk/environment/role.tf

```diff
@@ -45,7 +45,7 @@ resource "aws_iam_role_policy_attachment" "describe_environment" {
 }
 
 resource "aws_iam_policy" "describe_environment" {
-  name        = "DescribeEnvironment${join('', split('-', title(var.application)))}${title(var.environment)}"
+  name        = "describe-environmen-${var.application}-${var.environment}"
```

Comment:
Thanks, I had issues with this the other day, it was completely broken.

Still, I think you have a typo here, it should be `describe-environment`